lib/std/child_process.zig

```diff
@@ -423,6 +421,16 @@ pub const ChildProcess = struct {
             }
 
             if (self.cwd_dir) |cwd| {
+                // Remove the O_CLOEXEC flag. This is the only safe time to do it, between fork() and execve().
```

Comment:
Huh? you don't need to set cloexec at all: you should `fchdir` and then `close` it.

Reply:
or actually: you shouldn't `close` it at all: they may want it to be inherited by the child.
==> just `fchdir` and let existing `O_CLOEXEC` flag do the work.